ci/Dockerfile-bun

```diff
@@ -33,8 +33,8 @@ ENV GOPATH=/opt/app-root/go \
     MAVEN_HOME="/opt/maven/${MAVEN_VERSION}" \
     GRADLE_HOME="/opt/gradle/${GRADLE_VERSION}" \
     SBT_HOME="/opt/sbt/${SBT_VERSION}" \
-    PYTHON_VERSION=3.12 \
-    PYTHON_CMD=/usr/bin/python${PYTHON_VERSION} \
+    PYTHON_VERSION=$PYTHON_VERSION \
+    PYTHON_CMD=/usr/bin/python3.12 \
```

Comment:
It might be possible to make this work without hardcoding 3.12. For some reason, it didn't work locally for me and I ended up wasting several hours to troubleshoot. Must be a quirk in nerdctl or dockerfile.